### PR TITLE
Bump schemas version to 1.0.0

### LINKED ISSUE #157: Release schemas version 1.0.0
### Is your proposal related to a problem?

No

### Describe the solution you'd like

Release version 1.0.0 of the NLP Sandbox schemas.

Depends on #153.

### PR DESCRIPTION
**Pull request by tschaffter**
Resolves #157

### CODE REVIEW

**Comment by tschaffter** [comment]
@cascadianblue I refactored the PHI Deidentifier app, mainly by creating one schema file per object. Also I collapsed the "strategy" schema and the "config" schema (now called "DeidentificationStep" to be more accurate and match the UI) into one schema. I'm interested in spending more time refactoring the specification of the PHI deidentifier after the launch of the project.

File: openapi/phi-deidentifier/components/schemas/DeidentifyResponse.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 type: object
-description: Response to deidentification request
+description: The response to deidentification request
```
Comment (cascadianblue) [inline openapi/phi-deidentifier/components/schemas/DeidentifyResponse.yaml:2]: This seems weird grammatically. I think it should either be "The response to a deidentification request" or "Response to deidentification request"

File: openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    format: float
+    description: The minimum confidence level for a given annotation to be de-identified
+    minimum: 0
+    maximum: 100
+    example: 42.4
+    default: 0
+  maskingCharConfig:
+    $ref: MaskingCharConfig.yaml
+  maskingAnnotationTypeConfig:
+    $ref: MaskingAnnotationTypeConfig.yaml
```
Comment (cascadianblue) [inline openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml:15]: I see why `annotationTypeConfig` might be unclear, but this doesn't seem like a clear name.

How about `AnnotationTypeMaskConfig` ?

File: openapi/phi-deidentifier/components/schemas/MaskingCharConfig.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+type: object
+description:
+  Configuration for the masking char strategy. E.g. "John Smith lives at 123
+  Main St" -> "********* lives at ***********".
+properties:
+  maskingChar:
+    type: string
+    description: Character used to mask annotated PII text.
```
Comment (cascadianblue) [inline openapi/phi-deidentifier/components/schemas/MaskingCharConfig.yaml:8]: Should this be PII or PHI?

File: openapi/commons/components/schemas/Tool.yaml [hunk 4]
```diff
@@ -7,39 +7,38 @@ properties:
...
     description: The URL to the homepage of the tool
     type: string
     format: uri
   toolType:
     $ref: ../parameters/ToolType.yaml
+  toolApiVersion:
+    description: The version of the tool OpenAPI specification
+    type: string
+    minLength: 1
+    pattern: "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```
Comment (thomasyu888) [inline openapi/commons/components/schemas/Tool.yaml:41]: This is an impressive pattern.  Is it for semantic versioning?

File: openapi/data-node/openapi.yaml [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
+openapi: 3.0.3
 info:
-  version: 0.3.1
+  version: 1.0.0
   title: NLP Sandbox Data Node API
   contact:
     name: The NLP Sandbox Team
     email: thomas.schaffter@sagebionetworks.org
-    url: 'https://nlpsandbox.github.io/nlpsandbox-schemas'
+    url: https://nlpsandbox.io
```
Comment (thomasyu888) [inline openapi/data-node/openapi.yaml:8]: The site is currently down?

**Review by thomasyu888 — APPROVED** [review]

File: openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  redactConfig:
+    $ref: RedactConfig.yaml
+  dateOffsetConfig:
```
[the remark this answers is not in the document]
Comment (tschaffter) [reply openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml:18]: Sounds good, thanks!

File: openapi/phi-deidentifier/components/schemas/DeidentifyResponse.yaml (hunk 1 above)
Comment (tschaffter) [reply openapi/phi-deidentifier/components/schemas/DeidentifyResponse.yaml:2]: Fixed

File: openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml (hunk 2 above)
Comment (tschaffter) [reply openapi/phi-deidentifier/components/schemas/DeidentificationStep.yaml:15]: OK

File: openapi/phi-deidentifier/components/schemas/MaskingCharConfig.yaml (hunk 3 above)
Comment (tschaffter) [reply openapi/phi-deidentifier/components/schemas/MaskingCharConfig.yaml:8]: Fixed

File: openapi/commons/components/schemas/Tool.yaml (hunk 4 above)
Comment (tschaffter) [reply openapi/commons/components/schemas/Tool.yaml:41]: Yes. See https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

File: openapi/data-node/openapi.yaml (hunk 5 above)
Comment (tschaffter) [reply openapi/data-node/openapi.yaml:8]: We still need to setup the redirection to the Synapse production site.

**Pull request merged**